ledger/src/sigverify_shreds.rs

```diff
@@ -46,7 +46,11 @@ fn verify_shred_cpu(packet: &Packet, slot_leaders: &HashMap<u64, [u8; 32]>) -> O
     let slot_start = sig_end + size_of::<ShredType>();
     let slot_end = slot_start + size_of::<u64>();
     let msg_start = sig_end;
-    let msg_end = packet.meta.size;
+    let msg_end = if packet.meta.repair {
```

Comment:
I'd rather we do this detection based on slot.   At slot X on mainnet-beta and testnet, we hard cut over to the new shred size.

Reply:
ah yeah totally, so we can roll out at some slot `current + X` What's the standard practice for the distance `X` here?

Reply:
Yeah two values for v1.1 and v1.0 respectively.  We've used +500k-1m slots before, depending on the urgency of the update.